numpy/linalg/tests/test_linalg.py

```diff
@@ -2002,3 +2002,10 @@ def test_unsupported_commontype():
     arr = np.array([[1, -2], [2, 5]], dtype='float16')
     with assert_raises_regex(TypeError, "unsupported in linalg"):
         linalg.cholesky(arr)
+
+def test_issue_13401():
```

Comment:
Only this & one other linalg test fail with SkylakeX emulation, even in full mode, which surprised me.